Makefile

```diff
@@ -382,7 +383,7 @@ export RELEASE_BODY
 ## Pushes a github release and release artifacts produced by `make release-build`.
 release-publish: release-prereqs $(UPLOAD_DIR)
 	# Push the git tag.
-	git push origin $(CALICO_VER)
+	git push $(GIT_UPSTREAM) $(CALICO_VER)
```

Comment:
Maybe we can add checking that GIT_UPSTREAM is not empty... Maybe check it in `release-prereqs`?